### ISSUE TITLE
Casing issue breaks on Linux

### ISSUE THREAD

**Issue by craigs100** [DanDiplo/Umbraco.GodMode#22]
When running on Linux, when opening God Mode in the Settings section there is an error:
/App_Plugins/DiploGodMode/backoffice/godModeTree/intro.html

Think it should be:-
/App_Plugins/DiploGodMode/BackOffice/GodModeTree/intro.html

I guess it's a command decision as to which way round you change it. Either the call or the folder names.

**Comment by DanDiplo** [comment]
Hi Craig. I've just pushed an update which I _hope_ will fix this, though I haven't been able to test (I will really need to get around to installing WSL!).

It's on NuGet now: https://www.nuget.org/packages/Diplo.GodMode/9.1.0

It also adds some extra diagnostics and a new Tag Browser function, too. 

If you still have issues after the update I'd be really grateful if you could help me diagnose them. For instance, if you can fix it by renaming a file in `App_Plugins` then let me know.

**Comment by nul800sebastiaan** [comment]
There's still one more thing in the current version that need renaming:

![image](https://user-images.githubusercontent.com/304656/135981941-e5b71439-1f88-4d56-8f97-21e9de4fac62.png)

`backoffice` to `BackOffice`

Then after that it would be good for someone on Linux to check all the screens and see if they work properly. Maybe you want to add a `help-wanted` label here @DanDiplo so someone can pick it up as part of Hacktober?

**Comment by DanDiplo** [comment]
@nul800sebastiaan Yeah, when I went through the code I noticed some more. I've made some more fixes now, but just need help in testing them. I've released a new `9.2.0` version for 9 with the fixes and some new additions for the Template browser... I've also taken your good advice and added the "help wanted" tag - great idea! 👍 

There's also another issue I could do with help on, too - not sure if it's me or an Umbraco issue - https://github.com/DanDiplo/Umbraco.GodMode/issues/23
